### PR TITLE
chore: batch update deps

### PR DESCRIPTION
**Pull request by codyoss**
go get -d google.golang.org/api@v0.54.0
        go get -d google.golang.org/grpc@v1.40.0
        go get -d cloud.google.com/go@v0.93.3

### CODE REVIEW

File: containeranalysis/go.mod [hunk 1]
```diff
@@ -3,11 +3,12 @@ module cloud.google.com/go/containeranalysis
 go 1.16
 
 require (
-	cloud.google.com/go v0.92.1
+	cloud.google.com/go v0.93.3
+	cloud.google.com/go/grafeas v0.0.0-20210817223811-71387f0142a4
```
Comment (tbpg) [inline containeranalysis/go.mod:7]: Should this be tagged?

**Comment by codyoss** [comment]
This supersedes #4639 4620

**Review by tbpg — APPROVED** [review]
One comment, otherwise LGTM.

**Comment by codyoss** [comment]
@hengfengli FYI about the test that I needed to update. I don't believe this should have an impact on the actual client.

**Pull request merged**